### PR TITLE
css: fixed dashboard filter row

### PR DESCRIPTION
**Pull request by thelostone-mc**
###### Before

screen shot 2018-05-29 at 1 31 11 pm

###### After

screen shot 2018-05-29 at 1 33 05 pm

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- html

### CODE REVIEW

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -324,11 +324,11 @@ var process_stats = function(results) {
       break;
     case 1:
       matchesEl.html(num + gettext(' Matching Result'));
-      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-body'>(" + stats + ')</span>');
+      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-caption'>(" + stats + ')</span>');
```
Comment (SaptakS) [inline app/assets/v2/js/pages/dashboard.js]: I know it's not your fault but I think `')</span>'` to `")</span>"` keeps the uniformity.

Comment (thelostone-mc) [reply app/assets/v2/js/pages/dashboard.js]: Nice catch

File: app/assets/v2/js/pages/dashboard.js [hunk 2]
```diff
@@ -324,11 +324,11 @@ var process_stats = function(results) {
       break;
     case 1:
       matchesEl.html(num + gettext(' Matching Result'));
-      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-body'>(" + stats + ')</span>');
+      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-caption'>(" + stats + ")</span>");
       break;
     default:
       matchesEl.html(num + gettext(' Matching Results'));
-      fundingInfoEl.html("<span id='modifiers'>Funded Issues</span><span id='stats' class='font-body'>(" + stats + ')</span>');
+      fundingInfoEl.html("<span id='modifiers'>Funded Issues</span><span id='stats' class='font-caption'>(" + stats + ")</span>");
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/js/pages/dashboard.js]: @SaptakS / @mbeacom 
Do we stick to single quotes or update stickler ?

**Comment by owocki** [comment]
meta thought - we need to figure out how to scale the dashboard to many many many more tokens being used

**Pull request merged**